mis_builder/models/mis_report.py

```diff
@@ -147,18 +147,17 @@ class MisReportKpi(models.Model):
 
     _order = "sequence, id"
 
-    def name_get(self):
-        res = []
+    @api.depends("description")
+    def _compute_display_name(self):
         for rec in self:
-            name = f"{rec.description} ({rec.name})"
-            res.append((rec.id, name))
-        return res
+            rec.display_name = f"{rec.description} ({rec.name})"
+            return rec
```

Comment:
There should be no need to return rec here.

Reply:
Return is needed so we can compare the domain and the return of the function:
`return self.search(domain, limit=limit)._compute_display_name()`

Reply:
That does not sound correct. A compute method should not return a value. See my other comments.